lib/default.nix

```diff
@@ -71,8 +71,8 @@ let
     inherit (lists) singleton foldr fold foldl foldl' imap0 imap1
       concatMap flatten remove findSingle findFirst any all count
       optional optionals toList range partition zipListsWith zipLists
-      reverseList listDfs toposort sort take drop sublist last init
-      crossLists unique intersectLists subtractLists
+      reverseList listDfs toposort sort compareLists take drop sublist
+      last init crossLists unique intersectLists subtractLists
```

Comment:
One of the intentions behind making the inherits explicit here is so we don't need to make all the lib functions available in the top level, but instead say `lib.lists.compareLists`. If this seems like a reasonable thing to do, would you like to not `inherit` them here? It could be we're just not ready for that yet, seeing as I don't think many functions aren't exposed at the top level here.